Index: engine/__init__.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- engine/__init__.py	(date 1541780025000)
+++ engine/__init__.py	(date 1541780025000)
@@ -0,0 +1,0 @@
Index: engine/helper.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- engine/helper.py	(date 1541780336000)
+++ engine/helper.py	(date 1541780336000)
@@ -0,0 +1,0 @@
Index: .idea/AlgoRobinhood.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/AlgoRobinhood.iml	(date 1541780266000)
+++ .idea/AlgoRobinhood.iml	(date 1541780266000)
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="PYTHON_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="jdk" jdkName="Python 3.6" jdkType="Python SDK" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+  <component name="TestRunnerService">
+    <option name="PROJECT_TEST_RUNNER" value="Unittests" />
+  </component>
+</module>
\ No newline at end of file
Index: engine/constants.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- engine/constants.py	(date 1541780266000)
+++ engine/constants.py	(date 1541780266000)
@@ -0,0 +1,12 @@
+import requests
+
+Session = requests.Session()
+Session.headers = {
+    "Accept": "*/*",
+    "Accept-Encoding": "gzip,defalte,br",
+    "Accept-Language": "en-US,en;q=1",
+    "Content-Type": "application/x-www-form-urlencoded; charset=utf-8",
+    "X-Robinhood-API-Version": "1.0.0",
+    "Connection": "keep-alive",
+    "User-Agent": "Robinhood/823 (iphone; iOS 7.1.2, Scale/2.00)"
+}
